app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
+//const { handlePsqlError } = require("./controller/error.controller");
+const { controller_fetchTopics } = require("./controller/topic.controller");
+
+//app.use(express.json());
```

Comment:
Make sure comments don't make it onto Main branch unless they are essential messages for other devs reading your code.